modules/openapi-generator-gradle-plugin/src/main/kotlin/org/openapitools/generator/gradle/plugin/tasks/GenerateTask.kt

```diff
@@ -80,6 +80,15 @@ open class GenerateTask @Inject constructor(private val objectFactory: ObjectFac
     @Input
     val generatorName = project.objects.property<String>()
 
+    /**
+     * This is the configuration for reference paths where schemas for openapi generation are stored
+     * The schema target directory into which code will be generated
```

Comment:
Is this not the location from which we generate schema?

```suggestion
     * The directory which contains the additional schema files
```